robot_skills/src/robot_skills/util/entity.py

```diff
@@ -160,7 +160,7 @@ def __repr__(self):
 
 class PersonProperties(object):
     def __init__(self, name, age, emotion, gender, gender_confidence, pointing_pose, posture, reliability, shirt_colors,
-                 tags_dict, velocity, parent_entity):
+                 tags, tagnames, velocity, parent_entity):
```

Comment:
Why the change? Would always let keys become before values.

Reply:
The two things are different. It shouldn't have been a dict in the first place. In the legacy message definition a field named `tagnames` was defined but never used. Only `tags` is used. Once the message definition in people_recognition is cleaned up then `tagnames` should be removed completely.

Reply:
Add ToDo or add issue

Reply:
ToDo is added